src/common/string_utils.cpp

```diff
@@ -5,6 +5,26 @@
 namespace kuzu {
 namespace common {
 
+std::vector<std::string> StringUtils::splitComma(
+    const std::string& input, bool ignoreEmptyStringParts) {
+    auto result = std::vector<std::string>();
+    auto currentPos = 0u;
+    auto lvl = 0u;
+    while (currentPos < input.length()) {
+        if (input[currentPos] == '(') {
+            lvl++;
```

Comment:
Sometimes code coverage is not accurate. So just double check if this part is covered or not.